frontend/src/metabase-types/entities/common.ts

```diff
@@ -0,0 +1,9 @@
+import type { Collection } from "metabase-types/api";
+import type { IconName } from "metabase/core/components/Icon";
+
+export type WrappedEntity<Entity> = {
```

Comment:
All entity object types are `any`, so I need to create a new type that will extend the entity object and has common methods